code/gtts_player/vendor/gtts/tts.py

```diff
@@ -226,7 +226,7 @@ def _package_rpc(self, text):
 
         rpc = [[[self.GOOGLE_TTS_RPC, escaped_parameter, None, "generic"]]]
         espaced_rpc = json.dumps(rpc, separators=(",", ":"))
-        return "f.req={}&".format(urllib.parse.quote(espaced_rpc))
+        return f"f.req={urllib.parse.quote(espaced_rpc)}&"
```

Comment:
The files in vendor/ should be left alone

Reply:
Reverted it in 1347468.